app/dagger/MyApplicationLoader.java

```diff
@@ -2,43 +2,20 @@
 
 import play.Application;
 import play.ApplicationLoader;
-import play.Environment;
-import play.api.LoggerConfigurator$;
-import scala.compat.java8.OptionConverters;
-
-import java.util.Optional;
 
+/**
+ * This class loads an application through Dagger compile time dependency injection.
+ */
 public class MyApplicationLoader implements ApplicationLoader {
 
-    static class LoggerConfigurator {
-
-        private final play.api.LoggerConfigurator delegate;
-
-        public LoggerConfigurator(play.api.LoggerConfigurator delegate) {
-            this.delegate = delegate;
-        }
-
-        public static Optional<LoggerConfigurator> fromClassLoader(ClassLoader classLoader) {
-            return OptionConverters.toJava(LoggerConfigurator$.MODULE$.apply(classLoader)).map(LoggerConfigurator::new);
-        }
-
-        public void configure(Environment env) {
-            delegate.configure(env.underlying());
-        }
-    }
-
     @Override
-    public Application load(Context context) {
-        final ClassLoader classLoader = context.environment().classLoader();
-        final Optional<LoggerConfigurator> opt = LoggerConfigurator.fromClassLoader(classLoader);
```

Comment:
You still need to call the `LoggerConfigurator` here. See this example:

https://github.com/playframework/playframework/blob/3bd429d9520c297a037e41edaf7f5a6f77701852/documentation/manual/working/javaGuide/main/dependencyinjection/code/javaguide/di/components/CompileTimeDependencyInjection.java#L53-L64